### PR TITLE
Adding Promotion ELigibility

### PR DESCRIPTION
**Pull request by kswartz6**
Adding promotion eligibility samples.

# Description

Adding promotion eligibilities sample to SDK.

### CODE REVIEW

File: sdk/SdkSamples/SdkSamples.csproj [hunk 1]
```diff
@@ -45,21 +45,21 @@
     <Reference Include="Microsoft.Identity.Client, Version=4.31.0.0, Culture=neutral, PublicKeyToken=0a613f4dd989e8ae, processorArchitecture=MSIL">
       <HintPath>..\packages\Microsoft.Identity.Client.4.31.0\lib\net461\Microsoft.Identity.Client.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Store.PartnerCenter, Version=2.0.1.0, Culture=neutral, processorArchitecture=MSIL">
```
Comment (karlenHackForGood) [inline sdk/SdkSamples/SdkSamples.csproj:48]: Make sure to sync with master. 
image

File: sdk/SdkSamples/packages.config [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Identity.Client" version="4.31.0" targetFramework="net461" />
```
Comment (kaminasy) [inline sdk/SdkSamples/packages.config:3]: Same here

**Review by kaminasy — APPROVED** [review]

File: sdk/SdkSamples/packages.config (hunk 2 above)
Comment (kswartz6) [reply sdk/SdkSamples/packages.config]: Should be good now. Merged

File: sdk/SdkSamples/SdkSamples.csproj (hunk 1 above)
Comment (kswartz6) [reply sdk/SdkSamples/SdkSamples.csproj]: Should be good now after merge

File: sdk/SdkSamples/PromotionEligibilities/PostPromotionEligibilities.cs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    using Microsoft.Store.PartnerCenter.Models.PromotionEligibilities;
+    using Microsoft.Store.PartnerCenter.Models.PromotionEligibilities.Enums;
+
+    /// <summary>
+    /// A scenario that posts promotion eligibilities for a customer.
+    /// </summary>
+    public class PostPromotionEligibilities : BasePartnerScenario
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="CheckoutCart"/> class.
```
Comment (brijeshp-ms) [inline sdk/SdkSamples/PromotionEligibilities/PostPromotionEligibilities.cs:20]: PostPromotionEligibilities

Comment (kswartz6) [reply sdk/SdkSamples/PromotionEligibilities/PostPromotionEligibilities.cs]: Updated.

**Review by brijeshp-ms — APPROVED** [review]

**Pull request merged**